cluster-up/hack/common.sh

```diff
@@ -17,6 +17,8 @@ fi
 
 CDI_INSTALL_OPERATOR="install-operator"
 CDI_INSTALL_OLM="install-olm"
+CDI_OLM_MANIFESTS_CATALOG_SRC=""
```

Comment:
We really can't change anything in the 'cluster-up' directory as that is wiped and pulled when you we do a make generate.

Reply:
Oh good call.

Reply:
We currently have 4 fields in there already that need to go.

Reply:
@awels good call indeed! I moved all the variables to cluster-sync/install-config.sh. Please take  a look